### ISSUE TITLE
Change fragment URL part

### ISSUE THREAD

**Issue by sanjeev55** [zalando/skipper#2225]
Hello, 

Is there a way we can alter a hash route, for example i have a url as such:  ``<host>/#/home/user1`` i would like to alter this to something like this ``<host>/#/home/user2``. Is it at all possible to make changes like this using skipper filters? If not this maybe append to the existing hash route? 
Any information would highly appreciated.

**Comment by MustafaSaber** [comment]
Hi @sanjeev55 , If I understand you question correctly I think [`setPath`](https://opensource.zalando.com/skipper/reference/filters/#setpath) would achieve that.

**Comment by AlexanderYastrebov** [comment]
Hello, the `#` is called URL anchor or fragment identifier and is not sent to the server so it is not possible to modify it.
See https://stackoverflow.com/questions/3067491/is-the-anchor-part-of-a-url-being-sent-to-a-web-server

**Comment by szuecs** [comment]
You can change it in the response path for example in the location header, but otherwise see @AlexanderYastrebov response.

**Comment by sanjeev55** [comment]
Thank you for your reply @MustafaSaber @AlexanderYastrebov @szuecs 
But the `setPath` filter didn't work cause the path from ``/#/`` was not sent to the server side as @AlexanderYastrebov said. When I tried I got ``cannot GET /%23/home/user2``, even though there is a directory as such in the volume. I'll try changing the location header and observing the behavior. Thanks again.

**Comment by szuecs** [comment]
@sanjeev55 what do you want to achieve?
For me it's just guessing what you maybe want. As Alexander already wrote `#` lives in your browser and does not travel through the network in the request path. As I mention you can prepare a response link with that, such that your browser jumps to the anchor resource, but that's it.

**Comment by sanjeev55** [comment]
Hi @szuecs, thank you for your response. 
So what I am trying to achieve is to update that URL based on who the tenant is. Since I am using Keycloak the tenant information can be obtained from there. @AlexanderYastrebov said we cannot edit this particular URL from the skipper, so I figured I change the URL from react side itself. But before that, I will try to prepare the response link as you mentioned. 
Thanks again.

**Comment by sanjeev55** [comment]
So i was able to solve this by removing the hash router path and adding a normal path as `/api` from the react code and then in skipper I did the following: 

```
zalando.org/skipper-routes: |
      PathRegexp("/api") -> redirectTo(307, "<newhost>/#/home/workspace/tenant01") -> <shunt>
```
I have another ingress configured for the `<newhost>` which points to my desired service. For each tenant i have the similar configuration. I tried the setting up the location in the response header, but even though I could see the location being set properly it was not redirecting me to correct workspace. 

Thank you @szuecs @AlexanderYastrebov  for the help.
